Documentation/bpf/progtypes.rst

```diff
@@ -338,49 +338,88 @@ corresponding network driver name of an interface can be determined as follows:
+| Engleder               | tsne                    | >= 6.3      |
+|                        | (TSN Express Path)      |             |
++------------------------+-------------------------+-------------+
+| Freescale              | dpaa                    | >= 5.0      |
+|                        +-------------------------+-------------+
+|                        | dpaa2                   | >= 5.11     |
+|                        +-------------------------+-------------+
+|                        | enetc                   | >= 5.13     |
+|                        +-------------------------+-------------+
+|                        | fec_enet                | >= 6.2      |
++------------------------+-------------------------+-------------+
+| Fungible               | fun                     | >= 5.18     |
++------------------------+-------------------------+-------------+
+| Google                 | gve                     | >= 5.19     |
++------------------------+-------------------------+-------------+
+| Intel                  | ice                     | >= 5.5      |
+|                        +-------------------------+-------------+
+|                        | igb                     | >= 5.10     |
+|                        +-------------------------+-------------+
+|                        | igc                     | >= 5.13     |
+|                        +-------------------------+-------------+
+|                        | i40e                    | >= 4.13     |
+|                        +-------------------------+-------------+
+|                        | ixgbe                   | >= 4.12     |
+|                        +-------------------------+-------------+
+|                        | ixgbevf                 | >= 4.17     |
++------------------------+-------------------------+-------------+
+| Marvell                | mvneta                  | >= 5.5      |
+|                        +-------------------------+-------------+
+|                        | mvpp2                   | >= 5.9      |
+|                        +-------------------------+-------------+
+|                        | otx2                    | >= 5.16     |
++------------------------+-------------------------+-------------+
+| Mediatek               | mtk                     | >= 6.0      |
++------------------------+-------------------------+-------------+
+| Mellanox               | mlx4                    | >= 4.8      |
+|                        +-------------------------+-------------+
+|                        | mlx5                    | >= 4.9      |
++------------------------+-------------------------+-------------+
+| Microchip              | lan966x                 | >= 6.1      |
++------------------------+-------------------------+-------------+
+| Microsoft              | hv_netvsc (Hyper-V)     | >= 5.6      |
+|                        +-------------------------+-------------+
+|                        | mana                    | >= 6.0      |
```

Comment:
If I am not mistaken mana was added in v5.17 [see](https://github.com/torvalds/linux/commit/ed5356b53f070dea5dff5a01b740561cb8222199)